packages/fiori/src/Wizard.js

```diff
@@ -364,6 +398,132 @@ class Wizard extends UI5Element {
 	 */
 	onResize() {
 		this.width = this.getBoundingClientRect().width;
+
+		if (this.responsivePopover && this.responsivePopover.opened) {
+			this._closeRespPopover();
+		}
+	}
+
+	_adjustHeaderOverflow() {
```

Comment:
add some comments and try to split this a little bit.

Reply:
> add some comments and try to split this a little bit.

Done